lib/dishes.rb

```diff
@@ -0,0 +1,15 @@
+class Dishes
+
+	attr_accessor :type, :price
+
+	def initialize(type, price)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `type`. If it's necessary, use `_` or `_type` as an argument name to indicate that it won't be used.